setup.cfg

```diff
@@ -1,24 +1,125 @@
+[metadata]
+name = woodwork
+version = 0.14.0
+author = Alteryx Inc.
+license = BSD 3-clause
+long_description = file: READEME.md
+url = https://github.com/alteryx/woodwork/
+long_description_content_type = 'text/markdown'
+author_email = open_source_support@alteryx.com
+keywords = data science machine learning typing
+description = a two-dimensional data object with labeled axes and typing information
+
+classifiers =
+    Development Status :: 3 - Alpha
+    Intended Audience :: Developers
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
Specify operating systems like in Featuretools?  https://github.com/alteryx/featuretools/blob/aaa84e3634988b2e6d2c528ff240a397655024d7/setup.cfg#L24

Reply:
updated